PERMISSION_FIX.md

```
@@ -0,0 +1,293 @@
+```
+
+2. **Decode tại https://jwt.io**
+
+3. **Kiểm tra payload:**
+```json
+{
+  "sub": "user@example.com",
+  "roles": ["USER"],           // ❌ Thiếu
+  "permissions": ["COURSE_READ"], // ❌ Thiếu
+  "exp": 1234567890
+}
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)